roles/tuned/templates/recommend.conf

```diff
@@ -1,11 +1,11 @@
 [openshift-control-plane,master]
-/etc/origin/master/master-config.yaml=.*
+/etc/sysconfig/atomic-openshift-node=.*BOOTSTRAP_CONFIG_NAME.*master
```

Comment:
`BOOTSTRAP_CONFIG_NAME` is not guarenteed to be present in sysconfig settings - its gone after 3.10 -> 3.11 upgrade for instance

Reply:
Why not leaving just `node-config.yaml`? BOOTSTRAP_CONFIG_NAME is used just once during initial setup and isn't applicable during node run - its would use `node-config.yaml`. sysconfig would be reset during 3.10 -> 3.11 upgrade and won't have any config mentioned.

The original issue has been filed earlier than master/infra/compute were written into node-config, but currently it would certainly be written there. Also, `/etc/origin/node/node-config.yaml` would appear both in OKD and OCP - so its seems like a natural choice to me

Reply:
Because node-config.yaml is not populated by the node labels long after the "Node Bootstrap Preparation" is complete.  Even tasks like GlusterFS install start before the labels are set in node-config.yaml.  That's why I moved the location I call the "tuned" role and used atomic-openshift-node, as this is the file the node role is visible first.  I'd love to use node-config.yaml exclusively for this (see https://bugzilla.redhat.com/show_bug.cgi?id=1569917), but it looks this is simply not possible at present with major changes.  Any other suggestions for fixing this cleanly and quickly welcome.  In the future, the planned fix will be containerized tuned watching node labels changes.  Thanks for the review, btw!

Reply:
We also cannot rely on the configmap name matching these strings. They match in our defaults, but it's perfectly valid to name your configmap `node-config-sdodson` and that configmap applies master labels.